pkg/store/proxy.go

```diff
@@ -104,15 +123,69 @@ func (s *ProxyStore) Info(ctx context.Context, r *storepb.InfoRequest) (*storepb
 	res.MaxTime = MaxTime
```

Comment:
I know it is not yours, but can we fix above code? It looks weird that we loop over `range store` then check `len(stores)` 2 times. 

Also `MaxTime -> maxTime`, `MinTime ->  minTime`

Reply:
Moved all `len(stores) == 0` to an early return before anything else.